applications/openbrush.md

```diff
@@ -0,0 +1,273 @@
+| 1b.    | Multi token(Erc1155)             | We will implement reusable [Erc1155](https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/contracts/token/ERC1155) analog on ink!                                                                                                         |
+| 2a.    | AccessControl                    | We will implement reusable [AccessControl](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/AccessControl.sol) analog on ink!                                                                                                         |
+| 2b.    | Ownable                          | We will implement reusable [Ownable](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable.sol) analog on ink!                                                                                     |
+
+
+#### Milestone 2 - Simplify usage of library. Add a new features which extend ink!. Provide macroses that will allow creation of your own base implementation.
+
+* **Estimated duration:** 6 weeks
+* **FTE:**  2.5
+* **Costs:** 29,999 USD
+
+| Number | Deliverable                                         | Specification                                                                                                                                                                                    |
+| -----  | -----------                                         | -------------                                                                                                                                                                                    |
+| 0a.    | License                                             | MIT                                                                                                                                                                                              |
+| 0b.    | Documentation                                       | We will enhance inline documentation, update previous documentation based on simplifications, documentation for new features.                                                                                        |
+| 0c.    | Testing Guide                                       | We will test macros, update tests according to new features, simplifications.                                                                                               |
+| 1.     | Remove boilerplate code                             | We will provide a macro which will allow to remove boilerplate during usage of library(Library provides implementation on rust level in internal trait. User must reuse internal implementation with ink! messages. Our macro will simplify it). It will simplify the code structure and usage. |
```

Comment:
Items 1-3 here were listed as achieved in the first version of your application. Could you clarify?

Reply:
Indeed. 
However, they are currently under reconstruction for a few reasons, mainly coz we found problems with the current approach. so they need to be optimized.

Reply:
I can go into details if you like